index.bs

```diff
@@ -0,0 +1,210 @@
+
+{{LayoutShift}} interface {#sec-layout-shift}
+=======================================
+
+<pre class="idl">
+    interface LayoutShift : PerformanceEntry {
+      readonly attribute long value;
+    };
+</pre>
+
+{{LayoutShift}} extends the following attributes of {{PerformanceEntry}} interface:
+
+* The {{PerformanceEntry/name}} attribute must return the {{DOMString}} <code>"layout-shift"</code>
+* The {{PerformanceEntry/entryType}} attribute must return <code>"layoutShift"</code>.
+* The {{PerformanceEntry/startTime}} attribute must return the value it was initialized to.
+* The {{PerformanceEntry/duration}} attribute must return 0.
+* The <dfn attribute for=LayoutShift>value</dfn> will return the value it is set to.
+
+A user agent implementing {{LayoutShift}} must perform the following steps:
+1. Run the <a>register a performance entry type</a> algorithm with <code>"layoustShift"</code> as input.
+
+Processing model {#sec-processing-model}
+========================================
+
+Modifications to the HTML specification {#sec-modifications-HTML}
+--------------------------------------------------------
+
+<em>This section will be removed once the <a href=https://html.spec.whatwg.org/multipage>HTML specification</a> has been modified.</em>
+
+For each <a>Element</a>, there is a:
+* <dfn>currentFrameStartingPoint</dfn> which is initially set to <code>null</code>.
+* <dfn>previousFrameStartingPoint</dfn> which is initially set to <code>null</code>.
+* <dfn>currentVisualRepresentation</dfn> which is initially set to <code>null</code>.
+* <dfn>previousVisualRepresentation</dfn> with is initially set to <code>null</code>.
+
+<div algorithm="additions to update rendering">
+    In the <a>update the rendering</a> step of the <a>event loop processing model</a>, add a step right after the step that calls <a>mark paint timing</a>:
+
+    1. For each fully active {{Document}} in <em>docs</em>, invoke the algorithm to <a>evaluate the layout instability</a> of that {{Document}}.
+</div>
+
+Evaluate Layout Instability {#sec-eval-layout-instability}
+--------------------------------------------------------
+
+<div algorithm="evaluate the layout instability">
+    When asked to <dfn export>evaluate the layout instability</dfn> given an active {{Document}} <var>doc</var>, run the following steps:
+
+    1. Let <var>elements</var> be the list of current elements in <var>doc</var>.
+    1. Let <var>unstable elements</var> be an empty list.
+    1. Let <var>total width</var> be the current <a>viewport</a> width.
+    1. Let <var>total height</var> be the current <a>viewport</a> height. 
+    1. For each <var>element</var> of <var>elements</var>:
+        1. Set <a>currentFrameStartingPoint</a> to the <a>starting point</a> of <var>element</var>.
```

Comment:
For all these mentions, these are associated concepts to an element. So it should be `Set |element|'s <a>currentFrameStartingpoint</a> to...`

Reply:
Done.